### DISCUSSION TITLE
Which model should I choose? Yolov8 Or Yolov8-worldv2 Or any other?

### DISCUSSION THREAD

**Discussion by rahuldev-2021** [ultralytics/ultralytics#15389] [Q&A]
I want to Implement object detection in real time and **one shot detection** 

The model should work on this following criteria on a custom dataset:
- less training data
- less computation 
- high accuracy 

With which model can I achieve this requirement for object detection in real time, I did some research about yolov8 and yolov8-worldv2, I kinda confused with which model to go ahead. 

Please do provide your inputs. 
Thank you..... :)

**Comment by glenn-jocher** [comment]
For your requirements of real-time object detection with less training data, less computation, and high accuracy, I recommend using YOLOv8-worldv2. This model is optimized for open-vocabulary detection, providing high efficiency and performance with reduced computational demands. It also supports deterministic training and easy export to other formats, making it a versatile choice for your custom dataset. If you encounter any issues, ensure you are using the latest package versions. For more details, you can refer to the [YOLO-World documentation](https://docs.ultralytics.com/models/yolo-world/).
